docs/testing.md

```diff
@@ -0,0 +1,28 @@
+# Testing Open Angel Arena
+This is the continuation of [Install guide](/docs/install.md)
+Here is a guide how you can modify and run the OAA
+
+## Preparation
+You first need to open dev tools and load OAA project
+
+## Building a map
+For game to run you also need a map,
+It is rebuilt by double-clicking `aaa.vmap` in `Asset browser` and then pressing `f9` and secting `Full compile` at top and `Build` at bottom of dialogue.
+`It should be done evey time you make changes to the map`
+
+## Running a game
+Now when you are set, you can start debugging your game, for this you need to open `dota console` in `Asset Browser`(Button on top)
```

Comment:
Uhm for me its `^`. You should say its the Button under `ESC`, left of `1` and above of `Tab`.